src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, OnInit, Input, ViewChild, HostListener }      from '@angular
 import { ActivatedRoute, Params } from '@angular/router';
 import { Location }               from '@angular/common';
 import { Router }            from '@angular/router';
+import { cloneDeep } from 'lodash';
```

Comment:
@joshuawilson I am getting a warning in VS code on this line saying '/Users/sudiptasen/workspace/almighty-ui/node_modules/@types/lodash/index.d.ts' is not a module. But it works fine after compilation (npm start). Any idea? Am I missing anything in the config?